tocount/params.py

```diff
@@ -6,7 +6,12 @@
 INVALID_TEXT_ESTIMATOR_MESSAGE = "Invalid value. `estimator` must be an instance of TextEstimator enum."
 INVALID_TEXT_MESSAGE = "Invalid value. `text` must be a string."
 
-LINEAR_MODELS = {
+TIKTOKEN_R50K_LINEAR_MODELS = {
     "english": {"a": 0.22027472695240083, "b": 1.3098454987590542},
-    "all": {"a": 0.24897308965467127, "b": 4.5430826510558830}
+    "all":     {"a": 0.24897308965467127, "b": 4.5430826510558830}
+}
```

Comment:
I see some changes that redo @sepandhaghighi 's PR, make sure your branch is updated to the latest `dev`